packages/material-ui/src/AccordionSummary/AccordionSummary.test.js

```diff
@@ -99,6 +111,17 @@ describe('<AccordionSummary />', () => {
     expect(button).not.to.have.class(classes.focused);
   });
 
+  it('should fire onBlur when the button blurs', () => {
```

Comment:
That one requires `act`.

Reply:
I don't understand why it's green in my env

Reply:
Ohhh, it's because I'm running the tests in watch mode! I still haven't got used to https://github.com/mochajs/mocha/issues/4347.

Reply:
It passes, but I think the idea is that we're asserting on the result of the field receiving focus and blurring and we want to ensure that the updates are completed prior to assertion.

There are a few other places in the codebase that are inconsistent about this, so I didn't wrap it here (e.g. `Select.test.js`, , but I probably should have.  Sorry about that.

@eps1lon what is the difference between this (in `blur should unset unfocused state`):

```
    act(() => {
      button.focus();
    });
    act(() => {
      button.blur();
    });
```

...and wrapping them with one `act`?  Is it that we're allowing `focus` to update before we `blur`?